content/docs/started/getting-started-k8s.md

```diff
@@ -71,6 +71,8 @@ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-ad
 
 ### Deploy Kubeflow
 
+In case you are deploying Kubeflow on multi-node cluster, you can follow this [guide](/docs/use-cases/kubeflow-on-multinode-cluster) to set up your system to use remote NFS filesystem in cluster nodes.
```

Comment:
This looks good! I have just one suggestion: Change "In case " to "If".

I'll leave it up to @alecglassford to give the final LGTM on the content.

Reply:
+1 to Sarah's comment. I would also add a couple of `a`s:

`a multi-node cluster`
`a remote NFS filesystem`